mkt/webapps/indexers.py

```diff
@@ -297,6 +297,8 @@ def extract_document(cls, pk=None, obj=None):
         d['is_rereviewed'] = is_rereviewed
         d['rereview_date'] = (obj.rereviewqueue_set.get().created
                               if is_rereviewed else None)
+        is_homescreen = obj.tags.filter(tag_text=mkt.HOMESCREEN).exists()
+        d['is_homescreen'] = is_homescreen
```

Comment:
Would be nice to start adding tests to `mkt/webapps/tests_indexers.py` - it does not cover much right now, but the more we add the quicker coverage will improve :)